distributed/scheduler.py

```diff
@@ -282,78 +283,188 @@ class WorkerState:
 
     # XXX need a state field to signal active/removed?
 
+    _actors: set
+    _address: str
+    _bandwidth: double
+    _extra: dict
+    _has_what: set
+    _hash: Py_hash_t
+    _last_seen: double
+    _local_directory: str
+    _memory_limit: Py_ssize_t
+    _metrics: dict
+    _name: object
```

Comment:
This is an `object` as it could be a `str` or an `int`. See discussion in PR ( https://github.com/dask/distributed/pull/4296 ) for more context.